src/render/square-tilemap.vert

```diff
@@ -27,11 +27,11 @@ void main() {
     vec2 position = Vertex_Position.xy;
     
     vec2 positions[4] = vec2[4](
-        vec2(position.x, position.y),
-        vec2(position.x, position.y + 1.0),
-        vec2(position.x + 1.0, position.y + 1.0),
-        vec2(position.x + 1.0, position.y)
-    );
+            vec2(position.x, position.y),
```

Comment:
Formatting looks a little off here.

Reply:
Yea, I updated my rotate branch, let me know what you think. (I also added another small comment for completeness)